pkg/installers/windows/Kovri32.iss

```diff
@@ -0,0 +1,90 @@
+; Kovri Installer for Windows, 32 bit variant
+; Copyright (c) 2013-2017, The Kovri I2P Router Project
+; See LICENSE
+
+[Setup]
+AppName=Kovri I2P Router
+AppVersion=Latest
+DefaultDirName={pf}\Kovri
+DefaultGroupName=Kovri I2P Router
+UninstallDisplayIcon={app}\Kovri.ico
+PrivilegesRequired=admin
+; ArchitecturesInstallIn64BitMode=x64
+; ArchitecturesAllowed=x64
+WizardSmallImageFile=WizardSmallImage.bmp
+WizardImageFile=WizardImage.bmp
+DisableWelcomePage=no
+LicenseFile=LICENSE
+OutputBaseFilename=KovriSetup32
+
+
+[Languages]
+Name: "en"; MessagesFile: "compiler:Default.isl"
+
+
+[Files]
+Source: "bin32\build\kovri.exe";      DestDir: "{app}"; Flags: comparetimestamp
+Source: "bin32\build\kovri-util.exe"; DestDir: "{app}"; Flags: comparetimestamp
+Source: "Kovri.ico";                  DestDir: "{app}"; Flags: comparetimestamp
+Source: "ReadMe.htm";                 DestDir: "{app}"; Flags: comparetimestamp
+
+; Install any new versions of "hosts.txt" and "publishers" in "\client\address_book"
+Source: "bin32\pkg\client\*"; DestDir: "{userappdata}\Kovri\client"; Flags: recursesubdirs comparetimestamp
+
+; Backup any existing user config files, as we will overwrite "kovri.conf" and "tunnels.conf" unconditionally
+; Note that Inno Setup goes through the "[Files]" entries strictly in the order given here,
+; therefore the old files are backed-up correctly BEFORE the new ones overwrite them
+Source: "{userappdata}\Kovri\config\kovri.conf";   DestDir: "{userappdata}\Kovri\config"; DestName: "kovri.conf.bak";   Flags: external skipifsourcedoesntexist 
+Source: "{userappdata}\Kovri\config\tunnels.conf"; DestDir: "{userappdata}\Kovri\config"; DestName: "tunnels.conf.bak"; Flags: external skipifsourcedoesntexist
+
+Source: "bin32\pkg\config\*"; DestDir: "{userappdata}\Kovri\config"; Flags: recursesubdirs ignoreversion
+
+
+
+[InstallDelete]
+; Delete .exe files that the "old" install batch file copied directly to the desktop,
+; as this installer works with another, user-selectable location for those files
+Type: files; Name: "{userdesktop}\kovri.exe"
+Type: files; Name: "{userdesktop}\kovri-util.exe"

```

Comment:
I think without resorting to programming, only with in-built script, there is probably no way to ask conditionally. Means: Setup will ask anyway, whether there are currently are any .exe files on the desktop or not. I implemented this, with default "Deleting? Yes", but please have a look at this when you test this weekend.